docs/infra/treatment-service.md

```diff
@@ -0,0 +1,132 @@
+# Treatment Service Deployment
+
+## Overview
+XP's Treatment Service is currently offered in two different versions:
+
+- a central Treatment Service, *deployed as a single deployment, to serve treatments to multiple 
+  clients/users with multiple projects*
```

Comment:
nit: `from multiple CaraML projects`

Reply:
`central` -> `standalone` may be more meaningful to describe the deployment mode. (Same comment about other occurrences.)

> to serve treatments to multiple clients/users with multiple projects

Shall we say "it can be configured to serve..." as it's not necessary to have a single standalone service only; if the user really wants they can have multiple standalone treatment services for each project.

Reply:
Added `CaraML` in and changed the description to 

```md
- a standalone Treatment Service, *it can be configured to serve treatments to multiple clients/users with multiple 
  CaraML projects*
```